test/src/js/config.spec.js

```diff
@@ -6,6 +6,6 @@ const underTest = require('../../../src/js/config');
 
 describe('./src/js/config', function () {
 	it('config should be an Object', function () {
-		expect(underTest).to.be.an('object');
+		expect(typeof underTest).to.equal('object');
```

Comment:
... as a result of fixing the polyfills, this test broke (presumably because in IE it gets a polyfill or something IDK).

Either way, seems like not an important enough test to warrant including broken polyfills 🤷‍♂️

Reply:
Wee-erd